src/canmatrix/canmatrix.py

```diff
@@ -1227,43 +1421,77 @@ def delSignal(self, signal):
                 for sig in signalList:
                     frame.signals.remove(sig)
 
-    def addSignalReceiver(self, frameName, signalName, ecu):
-        frames = self.globFrames(frameName)
+    def addSignalReceiver(self, framePattern, signalName, ecu):
+        """Add Receiver to all Frames by name pattern.
+
+        :param str framePattern: Frame name pattern
+        :param str signalName: signal name
+        :param BoardUnit ecu: Receiver ECU name
+        """
+        frames = self.globFrames(framePattern)
         for frame in frames:
             signal = frame.signalByName(signalName)
             if signal is not None:
                 signal.addReceiver(ecu)
                 for signal in frame.globSignals(signalName):
-                    signal.addReceiver(ecu)
+                    signal.addReceiver(ecu)  # already set, redundant line
```

Comment:
Huh.  This seems very explicit but yeah, I'm not sure the cases where `frame.globSignals(signalName)` would get additional signals.  @ebroecker?  See similar below as well.

Reply:
I could imagine, that the indentation is wrong here.
The Idea was to check first if there is a signal with the exact name.
Afterwards check if there is a signal searching with globSignals.

But this makes no sense.
There should only be the "globSignals" way to find the right signals, the "signalByName" could be removed?!

Reply:
Mainly I did not remarked the 2nd `addReceiver` runs on another object with the same name. I'll try to thing about it once againe.